### PR TITLE
chore: Suppress Conditon in April Network WAF

### PR DESCRIPTION
**Pull request by nschonni**
Related to #6104

### CODE REVIEW

**Comment by nschonni** [comment]
@sergey-shandar this should fix a bunch of the unrelated spelling failures on the new PRs

**Review by kpajdzik — APPROVED** [review]
LGTM. Waiting for @sergey-shandar's approval.

**Pull request merged**
